catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/XmlModelBuilder.java

```diff
@@ -324,6 +335,39 @@ public void setValue(JAXBElement<?> valueNode) {
     }
   }
 
+  private static class DistanceBufferSupplier extends TerminalNodeSupplier {
+    private Double distance = null;
+
+    DistanceBufferSupplier(final TerminalNodeSupplier original) {
+      super(original.reducer);
+      if (original.propertyNode != null) {
+        this.setProperty(original.propertyNode);
+      }
+      if (original.valueNode != null) {
+        this.setValue(original.valueNode);
+      }
+    }
+
+    public DistanceBufferSupplier setDistance(Double distance) {
+      this.distance = distance;
+      return this;
+    }
+
+    @Override
+    public JAXBElement<?> get() {
+      JAXBElement<?> reduced = super.get();
+      notNull(distance, "The Distance element cannot be created without a distance value");
+      if (!DistanceBufferType.class.equals(reduced.getDeclaredType())
+          || !(reduced.getValue() instanceof DistanceBufferType)) {
+        throw new FilterProcessingException(
+            "Incorrect use of DistanceBufferSupplier for given type " + reduced.toString());
+      }
+      DistanceBufferType value = (DistanceBufferType) reduced.getValue();
+      value.setDistance(new MeasureType().withUom("m").withValue(distance));
```

Comment:
❓ What is `"m"`? If it's significant enough then create a constant for it.

Reply:
`m` means the units of measure (UOM) are in meters. I'll make a constant for it